addons/website_sale/controllers/main.py

```diff
@@ -771,6 +771,9 @@ def cart(self, access_token=None, revive='', **post):
             order.order_line.filtered(lambda l: l.product_id and not l.product_id.active).unlink()
             values['suggested_products'] = order._cart_accessories()
             values.update(self._get_express_shop_payment_values(order))
+            if request.env.context.get('lang') != request.env['ir.http'].session_info().get('bundle_params', {}).get('lang'):
+                for line in order.order_line:
+                    line._compute_name()
```

Comment:
`_compute_name` handles multi records :)
```suggestion
                order.order_line. _compute_name()
```